packages/frontend-v2/pages/index.tsx

```diff
@@ -1,454 +1,190 @@
-        axios.delete(`http://localhost:3002/api/students/${uuid}`)
-      )
-    );
-    console.log("cleared students");
-  };
-
+const Header = (): JSX.Element => {
   return (
-    <>
-      <h1>GraduateNU Landing Page:</h1>
-      <div>
-        <h2>Testing api calls on backend v2</h2>
-        <p>All responses will be logged to the console</p>
-
-        <Button onClick={clearDataBase}>Clear DB</Button>
+    <HeaderContainer>
+      <Logo />
+      <Button size="sm">Sign In</Button>
+    </HeaderContainer>
+  );
+};
 
-        <div>
-          <h3>Auth Routes</h3>
+const Banner = (): JSX.Element => {
+  return (
+    <Box
+      pt={{ desktop: "5rem", laptop: "3rem", tablet: "2.5rem" }}
+      pb={{ desktop: "11rem", laptop: "9rem", tablet: "6.25rem" }}
+    >
+      <HStack
+        spacing={{ desktop: "6.25rem", laptop: "5rem", tablet: "3rem" }}
+        justifyContent="center"
+      >
+        <Image
+          boxSize={{ desktop: "34.25rem", laptop: "31.25rem", tablet: "25rem" }}
+          src="/husky.svg"
+          alt="husky"
+        />
+        <Flex w="35%" flexDirection="column" alignItems="center">
+          <Box>
+            <Heading
+              fontSize={{ desktop: "7xl", laptop: "6xl", tablet: "5xl" }}
+              color="primary.main"
+            >
+              Graduate
+            </Heading>
+            <Heading
+              fontSize={{ desktop: "7xl", laptop: "6xl", tablet: "5xl" }}
+              color="blue.700"
+            >
+              your way
+            </Heading>
+            <Text
+              pt="5%"
+              fontSize={{ desktop: "3xl", laptop: "2xl", tablet: "xl" }}
+              color="blue.700"
+            >
+              Navigate the Northeastern graduation requirements and create a
+              personalized plan of study.
+            </Text>
+          </Box>
           <Button
-            onClick={async () => {
-              const student = await API.auth.register({
-                fullName: "Aryan Shah",
-                email: "aryan1@gmail.com",
-                password: "aryan1234",
-                academicYear: 2019,
-                graduateYear: 2023,
-                catalogYear: 2019,
-                major: "Computer Science",
-                nuid: "000000000",
-              });
-              console.log(student);
-              setTokenInStorage(student.accessToken!);
-            }}
+            mr={{ desktop: "7.5rem", laptop: "6.25rem", tablet: "3.25rem" }}
+            mt="15%"
           >
-            Register
+            Get Started
           </Button>
+        </Flex>
+      </HStack>
+    </Box>
+  );
+};
 
-          <Button
-            onClick={async () => {
-              setTokenInStorage("");
-              console.log("Logged out, token reset");
-            }}
-          >
-            Logout
-          </Button>
+const Info = (): JSX.Element => {
+  const infoImageData = [
+    {
+      imageSource: "/landing_start.svg",
```

Comment:
seems like this `infoImageData` can be combined with `infoSectionData`. In this case, you can use one `map` down there and use react fragments `<></>` to combine the image and the description instead of having two maps.

Reply:
using the fragments made the order kinda weird, but i realized i could just make InfoSection (both image/text) its own component and use the single map... ty for the help :')))))